metadata/src/main/java/org/apache/kafka/metadata/migration/KRaftMigrationDriver.java

```diff
@@ -458,8 +458,9 @@ public void run() throws Exception {
                 zkMigrationClient.readAllMetadata(batch -> {
                     try {
                         if (log.isTraceEnabled()) {
-                            log.trace("Migrating {} records from ZK: {}", batch.size(), recordBatchToString(batch));
+                            log.info("Migrating {} records from ZK: {}", batch.size(), recordBatchToString(batch));
```

Comment:
It looks like you changed this for the purpose of debugging and forgot to change it back. BTW you can just change the test log4j.properties file to avoid this.

Reply:
This file shouldn't have been part of this commit at all. Thanks for catching this.